cmd/kube-apiserver/app/server.go

```diff
@@ -686,6 +688,11 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 				s.Authentication.ServiceAccounts.MaxExpiration > upBound {
 				return options, fmt.Errorf("the serviceaccount max expiration must be between 1 hour to 2^32 seconds")
 			}
+			// When service-account-extend-token-expiration is enabled during transition time, max expiration must be set
+			// long enough for fixing clients.
+			if s.Authentication.ServiceAccounts.ExtendExpiration && s.Authentication.ServiceAccounts.MaxExpiration < serviceaccount.ExpirationExtensionSeconds*time.Second {
+				return options, fmt.Errorf("serviceaccount max expiration must be longer than 1 year when service-account-extend-token-expiration is enabled")
```

Comment:
```suggestion
				return options, errors.New("serviceaccount max expiration must be longer than 1 year when service-account-extend-token-expiration is enabled")
```

Reply:
I would rather keep it the same return format as code in the same file